mne_bids/write.py

```diff
@@ -203,6 +204,48 @@ def _events_tsv(events, raw, fname, trial_type, overwrite=False,
     _write_tsv(fname, data, overwrite, verbose)
 
 
+def _readme(kind, fname, overwrite=False, verbose=True):
+    """Create a README file and save it.
+
+    This will append a citation for mne bids if the README exists and does
+    not have a citation. Otherwise a new file will be created with the
+    appropriate citations and methods paragraph.
```

Comment:
```suggestion
    This will write a README file containing an MNE-BIDS citation.
    If a README already exists, the behavior depends on the 
    `overwrite` parameter, as described below.
```